include/util.hpp

```diff
@@ -14,7 +14,10 @@
 namespace hypha
 {
     eosio::checksum256 getRoot(const eosio::name &contract);
+    eosio::checksum256 getDAO(const eosio::name &dao_name);
+
     ContentGroups getRootContent(const eosio::name &contract);
+    ContentGroups getDAOContent(const eosio::name &dao_name);
     eosio::asset adjustAsset(const eosio::asset &originalAsset, const float &adjustment);
```

Comment:
Previously, the ```dao_name``` == ```contract```, or ```dao.hypha```.  We will still need a contract-level root document for data like the smart contract and client semantic versions, but dho-level settings will hang off of the ```dho``` doc.